src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -92,29 +91,49 @@ class CreateCourseForm extends React.Component {
                 required
               />
               <Field
-                name="coursePrice"
+                name="price"
                 component={RenderInputTextField}
                 type="number"
                 label={
                   <React.Fragment>
-                    Course price (USD)
+                    Price (USD)
                     <span className="required">*</span>
                   </React.Fragment>
                 }
                 required
               />
+              <h3>First run of your Course</h3>
+              <hr />
+              <div>
+                Start Date
+                <span className="required">*</span>
+              </div>
+              <Field
+                name="start"
+                type="date"
+                component="input"
+                id="start-date"
+                label="Start Date"
```

Comment:
Why a different label pattern here? In the rest of the Fields you pass in a React.Fragment etc. But here, you've put that bit above the Field.

Reply:
You should be able to still use RenderInputTextField and pass in type="date". It's the same style as price